guava/src/com/google/common/base/Ascii.java

```diff
@@ -544,11 +546,24 @@ public static boolean isUpperCase(char c) {
    *     truncationIndicator}
    * @since 16.0
    */
-  public static String truncate(CharSequence seq, int maxLength, String truncationIndicator) {
+  @SuppressWarnings({
+    /*
+     * The parameter maxLength should be annotated @GTEqLengthOf("#3").
```

Comment:
Add a note that `@GTEqLengthOf` does not exist and that's why you have not annotated it that way.